inference-engine/src/transformations/include/transformations/op_conversions/gather_normalize_negative_indices.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <transformations_visibility.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+    class TRANSFORMATIONS_API GatherNegativeConstIndicesNormalize;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief GatherNegativeConstIndicesNormalize check if indices value is negative scalar and
+ * normalize it using ShapeOf->Add->Cast subgraph.
+ * We need to remove this transformation after support a negative indices in
```

Comment:
after adding support of negative indices ...

Reply:
Fixed.